Introduce idempotent support for kafka output (#52382)
elastic/beats: docs/reference/auditbeat/kafka-output.md

Note:`queue` options can be set under `auditbeat.yml` or the `output` section but not both.

### `idempotent` [_idempotent] 

{applies_to}`stack: ga 9.6.0`

> When Idempotent is enabled, the producer will ensure that exactly one copy of each message is written. 

Reviewer: ```suggestion
When Idempotent is enabled, the producer ensures that exactly one copy of each message is written. 
```

We don't use future tense in the docs 🙂 Please apply to all files 🙏

> This setting requires kafka version to be >=0.11.0.0, `max_retries` to be greater than `0` and `required_acks` to be equal to `-1`.   

Reviewer: ```suggestion
This setting requires kafka version to be >=0.11.0.0, `max_retries` to be greater than `0`, and `required_acks` to be equal to `-1`.   
```

> The default value is false

Reviewer: ```suggestion
The default value is `false`.
```

Could you please change that on every line where this repeats?
